src/styles/mediaQueries.js

```
@@ -15,6 +15,11 @@ const mediaQueryTabletLandscapeOnly = (content) => css`
     ${content};
   }
 `
+const mediaQueryTabletLandscapeUp = (content) => css`
```

ESLint

[error] 'mediaQueryTabletLandscapeUp' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)